src/libstd/sys/windows/process.rs

```diff
@@ -90,6 +90,12 @@ impl Command {
     pub fn cwd(&mut self, dir: &OsStr) {
         self.cwd = Some(dir.to_os_string())
     }
+    pub fn get_program(&self) -> &OsString {
+        &self.program
+    }
+    pub fn args_iter(&self) -> slice::Iter<OsString> {
```

Comment:
It may be best to delegate the `Debug` implementation to each platform's module in that case, we try to avoid differences in APIs wherever possible across the platforms.